### DISCUSSION TITLE
Getting Import error in Ubuntu and not able to install package using apt-get

### DISCUSSION THREAD

**Discussion by thirtha-prasad** [pemistahl/lingua-py#128] [Q&A]
Getting below error, Package in installed using PIP and available in the python path.

ImportError: cannot import name 'Language' from 'lingua' (/usr/local/lib/python3.9/dist-packages/lingua/__init__.py)

**Comment by pemistahl** [comment]
Hi, 

I don't have an Ubuntu system for testing. So you just installed it with `pip install lingua-language-detector`? Because in the title you say something about apt-get. Did you try to install Python with apt-get? My guess is that your Python installation does not work as expected. The library should work fine for every Python installation >= 3.8 on every operating system.

Can you please show me your exact code where you try to import the library? Thank you.

**Reply by thirtha-prasad** [reply]
Error: 
root@362e9d939caa:~/project# python3 pyUtil/detectLanguage.py cypress/results/portfolio/url-0/en_strings.json cypress/results/portfolio/url-0/en_results.csv
Traceback (most recent call last):
  File "/root/project/pyUtil/detectLanguage.py", line 1, in <module>
    from lingua import Language, LanguageDetectorBuilder, IsoCode639_1, IsoCode639_3
ImportError: cannot import name 'Language' from 'lingua' (/usr/local/lib/python3.9/dist-packages/lingua/__init__.py)
root@362e9d939caa:~/project#


I have installed python3.9 using command "apt-get install python3" and pip3 by " apt-get install python3-pip"

And installed lingua by command "pip3 install lingua-language-detector"

My import code is: from lingua import Language, LanguageDetectorBuilder

**Reply by pemistahl** [reply]
This should work perfectly fine. I don't see a reason why the import should fail. I'm sorry, I don't have any idea.

**Reply by thirtha-prasad** [reply]
I am using the docker below docker image 
    docker:
      - image: cypress/browsers:node18.6.0-chrome105-ff104
Is it possible to take a look.

**Reply by pemistahl** [reply]
Then it's probably a docker issue. I cannot help you with that, sorry.

**Reply by thirtha-prasad** [reply]
This issue is there in any Linux machine, I tried with other Linux images also and end up with same issue.

circleci@e2c945066b71:~$ python project/pyUtil/detectLanguage.py
Traceback (most recent call last):
  File "/home/circleci/project/pyUtil/detectLanguage.py", line 1, in <module>
    from lingua import Language, LanguageDetectorBuilder, IsoCode639_1, IsoCode639_3
ImportError: cannot import name 'Language' from 'lingua' (/home/circleci/.pyenv/versions/3.9.16/lib/python3.9/site-packages/lingua/__init__.py)

**Reply by thirtha-prasad** [reply]
Other installed python packages are able to import correctly.

**Comment by Venkatesh-inviz-UpperFunnel** [comment]
Any update or fix found ,I am facing same issue,Seeking help

**Reply by pemistahl** [reply]
Have you installed the wrong package? On PyPI, there are two packages named `lingua` and `lingua-language-detector`. The former is wrong, the latter is correct. If you have installed both, then please uninstall the former package again.

**Reply by Venkatesh-inviz-UpperFunnel** [reply]
Followed above steps,Still facing import error..
>>> from lingua import Language, LanguageDetectorBuilder
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'Language' from 'lingua' (unknown location)

**Reply by Venkatesh-inviz-UpperFunnel** [reply]
I am trying to import like below @pemistahl 
from lingua import Language, LanguageDetectorBuilder

**Reply by pemistahl** [reply]
I don't have a Linux machine, so I cannot try myself. Perhaps this Stack Overflow post can help you:

https://stackoverflow.com/questions/17083595/python-importerror-while-module-is-installed-ubuntu

**Comment by mikeyang01** [comment]
I got same error, solving it by uninstall both "lingua-language-detector" and "lingua", and install "lingua-language-detector" again, works
